tests/augmentations/test_sepia.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+import numpy as np
+
+from discolight.annotations import (annotations_to_numpy_array)
+from discolight.augmentations.sepia import Sepia
+
+def test_my_augmentation_sepia():
+    img =  np.array(([
+        [[ 62, 238, 229],
+         [242,  39, 182]],
+        [[167, 153,  41],
+         [130,  63, 162]]]))
+    
+    annotations = []
+    
+    expected_red = np.array([[250, 159], [191, 130]])
+    expected_green = np.array([[223, 141], [170, 115]])
+    expected_blue = np.array([[173, 110], [132, 90]])
+    expected_aug_img = np.zeros(img.shape, dtype=np.uint8)
+    expected_aug_img[:, :, 0] = expected_red
+    expected_aug_img[:, :, 1] = expected_green
+    expected_aug_img[:, :, 2] = expected_blue
+    
+    bboxes = annotations_to_numpy_array(annotations)
+    
+    augmentation = Sepia()
+    
+    aug_img, aug_bboxes = augmentation.augment(img.copy(), bboxes.copy())
+    print(aug_img, flush=True)
+    print(expected_aug_img, flush=True)
+    assert np.allclose(
+        expected_aug_img, aug_img
+    ), "Performing augmentation does not yield expected image"
+    assert np.allclose(
```

Comment:
This is kind of splitting hairs, but since a `ColorAugmentation` is really supposed to leave the bounding boxes untouched, the better test might be `np.array_equal` rather than just `np.allclose`.